test/modules/TestResultCoordinator/Settings.cs

```diff
@@ -62,7 +62,15 @@ class Settings
             this.DurationBeforeVerification = verificationDelay;
             this.ConsumerGroupName = "$Default";
             this.StorageAccountConnectionString = Preconditions.CheckNonWhiteSpace(storageAccountConnectionString, nameof(storageAccountConnectionString));
-            this.NetworkControllerType = GetNetworkControllerType(networkControllerRunProfileName);
+            this.NetworkControllerType = this.GetNetworkControllerType(networkControllerRunProfileName);
+
+            this.TestInfo = testInfo.Split(",", StringSplitOptions.RemoveEmptyEntries)
+                                .Select(x => (KeyAndValue: x, SplitIndex: x.IndexOf('=')))
+                                .Where(x => x.SplitIndex >= 1)
+                                .ToDictionary(
+                                    x => x.KeyAndValue.Substring(0, x.SplitIndex),
+                                    x => x.KeyAndValue.Substring(x.SplitIndex + 1, x.KeyAndValue.Length - x.SplitIndex - 1));
```

Comment:
This is the same logic as we implemented in longhaul. Might be a good idea to consolidate it in edge util

Reply:
Agree, but not edge util; should be ModuleUtil project under test folder.